_posts/2018-03-28-building-coreboot-on-minnowboard.md

```diff
@@ -0,0 +1,244 @@
+Building coreboot on MinnowBoard
+================================
+
+Building coreboot on well supported platform such as Bay Trail is quite
+straightforward task, however we need to remember about some things in
+order to have coreboot working. First of all we need to provide up-to-date
+microcode and FSP (Firmware Support Package), which are not included in
+coreboot source tree and coreboot build system won't complain about it.
+Second thing is that Bay Trail has ME firmware on the same ROM as boot firmware
```

Comment:
@BartekLew truly it has TXE and I had discussion about that in [this thread](https://mail.coreboot.org/pipermail/coreboot/2018-March/086358.html).

Reply:
@pietrushnic :+1: